src/frontend/components/Chat/StudentChat/index.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { act, render, screen } from '@testing-library/react';
+import React from 'react';
+import { wrapInIntlProvider } from '../../../utils/tests/intl';
+import { useMessagesState } from 'data/stores/useMessagesStore';
+import { DateTime } from 'luxon';
+import { StudentChat } from '.';
+import { converse } from '../../../utils/window';
+import userEvent from '@testing-library/user-event';
+
+const sendMessageMock = (msgContent: string) =>
+  act(() =>
+    useMessagesState.getState().addMessage({
+      dateTime: DateTime.fromISO('2020-01-01T12:12:12'),
+      sender: 'John Doe',
+      content: msgContent,
+    }),
+  );
+
+jest.mock('../../../utils/window', () => ({
```

Comment:
```suggestion
jest.mock('utils/window', () => ({
```

Reply:
fix in [dc5678d7d921cc886ff643274d5110277b1a03e8](https://github.com/openfun/marsha/pull/1247/commits/dc5678d7d921cc886ff643274d5110277b1a03e8)